tools/spark/benchmark/README.md

```diff
@@ -24,6 +24,6 @@ Replace the cluster configuration file, the paths, spark.driver.scale, spark.dri
 We provided the power test scala script **[tpcds-power-test.scala](./scripts/tpcds-power-test.scala)** for users to run TPC-DS power test with Cloudtik cluster.
 Execute the following command to submit and run the power test script on the cluster,
 ```buildoutcfg
-cloudtik submit your-cluster-config.yaml $CLOUTIK_HOME/tools/spark/benchmark/scripts/tpcds-power-test.scala --conf spark.driver.scaleFactor=1 --conf spark.driver.fsdir="s3a://s3_bucket_name" --conf spark.driver.iterations=1 --jars \$HOME/runtime/benchmark-tools/spark-sql-perf/target/scala-2.12/spark-sql-perf_2.12-0.5.1-SNAPSHOT.jar
+cloudtik submit your-cluster-config.yaml $CLOUTIK_HOME/tools/spark/benchmark/scripts/tpcds-power-test.scala --conf spark.driver.scaleFactor=1 --conf spark.driver.fsdir="s3a://s3_bucket_name" --conf spark.sql.shuffle.partitions="\$(echo \$[\$(yarn node -list -showDetails | grep 'Configured Resources' | awk -F 'vCores:' '{print \$2}'  | awk -F '>' '{sum +=\$1}; END {print sum}') * 2])" --conf spark.driver.iterations=1 --jars \$HOME/runtime/benchmark-tools/spark-sql-perf/target/scala-2.12/spark-sql-perf_2.12-0.5.1-SNAPSHOT.jar
```

Comment:
Do you think it is good to provide some facilities (cloudtik commands) to get the key system information, such as the total of worker cores, the number of worker nodes?

For example, if you are on head, you can simple execute "cloudtik head info --num-worker-cpus",  "cloudtik head info --num-workers", ...

In this way, it will be simple either ask user to do this caculation or you call it in the above way much simpler?